extensions/2.0/Khronos/KHR_texture_ktx/README.md

```diff
@@ -0,0 +1,315 @@
+For instance, if model uses an atlasmap and the resulting baseColor, normal and orm textures are the same size and format they can be stored as different layers in the same KTX file.  
+This will then give a hint to the runtime implementation that these textures can be grouped together, potentially using less uniform samplers.  
+
+## glTF Schema Updates
+
+The `KHR_texture_ktx` extension is added to the `textures` object and specifies a `source` property that points to the index of the `image` which defines a reference to the KTX v2 image.
+
+The following glTF will load `image.ktx2`
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 1, 
+                    "layer": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.png"
+        },
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+When used in the glTF Binary (GLB) format the `image` that points to the KTX v2 resource uses the `mimeType` value of `image/ktx2`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "mimeType": "image/png",
+            "bufferView": 1
+        },
+        {
+            "mimeType": "image/ktx2",
+            "bufferView": 2
+        }
+    ]
+}
+```
+
+### Using Without a Fallback
+
+To use KTX v2 image without a fallback, define `KHR_texture_ktx` in both `extensionsUsed` and `extensionsRequired`. The `texture` object will then have its `source` property omitted as shown below.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "extensionsRequired": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+### JSON Schema
+
+[texture.KHR_texture_ktx.schema.json](schema/texture.KHR_texture_ktx.schema.json)
+
+## Color Texture Sources
+
+Texture data provided by this extension is encoded depending on the target format and material teture slot usage.    
+When this extension is used with material texture slots that expect sRGB-encoded values (RGB or RGBA format) and the texture format is 8 bits per pixel (VK_FORMAT_R8G8B8 or VK_FORMAT_R8G8B8A8) the texture uses sRGB encoding.  
```

Comment:
To align with the KTX spec, sRGB-encoded textures must use `VK_FORMAT_R8G8B8(A8)_SRGB` format and the appropriate DFD transfer function value. The subsequent sections mention only linear `_UNORM` variants.

Reply:
Thanks @lexaknyazev I have added SRGB formats for RGB and RGBA and updated the wording.
Also added 16 bit linear encoded formats for color texture slots.